package.json

```diff
@@ -75,6 +75,7 @@
   },
   "scripts": {
     "build": "lerna run build",
+    "watch": "support/dev.sh",
```

Comment:
sounds good to me.

I also realized on the :bike: home that we can get rid of the bash script entirely and just summon lerna directly :wind_chime:

Reply:
how can we pass arguments to `lerna run` though?

Reply:
like this: `npm run dev -- --scope @esri/* dev:umd --parallel`

e94b2d4

Reply:
That's clever, but I don't see how it's any better than `lerna run --scope @esri/* dev:umd --parallel`.

I'm brining bash script back.

Reply:
Also, I think the best API if you want to run it for all packages is `npm run dev -- umd`. The reason I had the arguments in the original order was b/c I suspected if one of them would ever end up being optional it would be the scope. I don't see a scenario where you need to  I think I'm going to put them back in that order, but worn't take the time to remember how you you write an `if` in bash right now.